### ISSUE TITLE
Missing arguments in azuredevops_project_pipeline_settings

### ISSUE THREAD

**Issue by leighmo** [microsoft/terraform-provider-azuredevops#765]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

We need to be able to set the following arguments in the Project pipeline settings as we leverage modules in other projects that reside outside of the existing project the pipeline is executed from. We also want to disable class build/release pipelines via code and have these checked regularly to make sure they are not being changed.
- "Limit job authorization scope to current project for release pipelines" - Most important
- "Disable creation of classic build and classic release pipelines" - Less important

Thanks

### New or Affected Resource(s)

azuredevops_project_pipeline_settings

### Potential Terraform Configuration

Untested

### References

No

**Comment by xuzhang3** [comment]
@leighmo  "Limit job authorization scope to current project for release pipelines"  has been support be resource [azuredevops_project_pipeline_settings.enforce_job_scope](https://registry.terraform.io/providers/microsoft/azuredevops/latest/docs/resources/project_pipeline_settings#enforce_job_scope)

**Comment by leighmo** [comment]
Hi @xuzhang3,

Thanks for the reply. There are two options in the Azure DevOps Pipeline settings. 

**Limit job authorization scope to current project for non-release pipelines**
Non-Release Pipelines can run with collection scoped access tokens unless this option is enabled. With this option enabled, you can reduce the scope of access for all non-release pipelines to the current project.

**Limit job authorization scope to current project for release pipelines**
Release pipelines can run with collection scoped access tokens unless this option is enabled. With this option enabled, you can reduce the scope of access for all release pipelines to the current project.

The current module covers off the first option "non-release pipelines" but does not address the second option "release pipelines"

![ADO-Pipeline-Settings](https://github.com/microsoft/terraform-provider-azuredevops/assets/33709703/3cbbbc51-259b-4e14-bae3-df33e53cb767)

Cheers

**Comment by ashleyghooper** [comment]
Yes, we've been disabling the limit for release pipelines as a manual step and would greatly appreciate being able to do this via TF.

**Comment by xuzhang3** [comment]
New features will be available after upgrading to v7 API

**Comment by xuzhang3** [comment]
Disable creation of classic build and classic release pipelines - This configuration cannot be updated via the API.
Issue: https://github.com/microsoft/azure-devops-go-api/issues/133

**Comment by citrix-pradeep** [comment]
Hi @xuzhang3 , can we have terraform support for  [azuredevops_project_pipeline_settings] - Support pipeline general settings for " Disable creation of classic build and classic release pipelines"
![image](https://github.com/microsoft/terraform-provider-azuredevops/assets/8372198/81af1328-6a08-4b09-bf7c-0eb78ab1e0ce)

**Comment by xuzhang3** [comment]
@citrix-pradeep SDK does not support these two properties yet.

**Comment by arodindev** [comment]
@xuzhang3 are there any plans on adding the missing project pipeline settings in near future? This would be very helpful for us.

**Comment by xuzhang3** [comment]
SDK update PR: https://github.com/microsoft/azure-devops-go-api/pull/193
